Automattic/wp-calypso
# AI Logo generator: Add initial components

Related to https://github.com/Automattic/jetpack/issues/34376 and https://github.com/Automattic/jetpack/issues/34379

## Proposed Changes

* Adds the initial components without functionality

## Testing Instructions
* Visit calypso home on a site (`/home/[site]`)
* Find the Create logo with AI among the quick links (likely, last one)
* Click and open the modal.

You should see the loader for a few seconds, followed by the first iteration on the logo presenter component.

* Check the code for any obvious problems
* Visual check, should match the Figma file

## Pre-merge Checklist
